.github/workflows/test.yml

```diff
@@ -13,12 +13,18 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    # set up conda
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
+      uses: conda-incubator/setup-miniconda@v2
       with:
-        python-version: ${{ matrix.python-version }}
+        auto-activate-base: true
+        activate-environment: ""
+    # install rdkit and openbabel
+    - name: Install rdkit
+      run: conda create -c conda-forge -n my-rdkit-env python=${{ matrix.python-version }} rdkit openbabel;
```

Comment:
I don't see where you use openbabel.

Reply:
Function `dpdata.rdkit.sanitize.convert_by_obabel()` will call obabel via command line. This is designed for the cases where the sanitization fails (using obabel to try to pre-process the mol).